mmv1/third_party/terraform/services/compute/resource_compute_instant_snapshot_test.go.tmpl

```diff
@@ -0,0 +1,159 @@
+package compute_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+
+	{{ if eq $.TargetVersionName `ga` }}
+		"google.golang.org/api/compute/v1"
+	{{- else }}
+		compute "google.golang.org/api/compute/v0.beta"
+	{{- end }}
+)
+
+func TestAccComputeInstantSnapshot_basicFeatures(t *testing.T) {
+	var is compute.InstantSnapshot
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	resource.Test(t, resource.TestCase{
```

Comment:
Should be `acctest.VcrTest` instead of resource.Test, this is causing the error in replaying.